spinalcordtoolbox/scripts/sct_propseg.py

```diff
@@ -622,7 +622,7 @@ def propseg(img_input, options_dict):
     fname_seg = os.path.join(folder_output, fname_out)
     fname_centerline = os.path.join(folder_output, os.path.basename(add_suffix(fname_data, "_centerline")))
     # in case header was rescaled, we need to update the output file names by removing the "_rescaled"
-    if rescale_header is not 1:
+    if rescale_header != 1:
         mv(os.path.join(folder_output, add_suffix(os.path.basename(fname_data_propseg), "_seg")),
                fname_seg)
         mv(os.path.join(folder_output, add_suffix(os.path.basename(fname_data_propseg), "_centerline")),
```

Comment:
Now, as for why the CI is failing... This is actually because the changes have uncovered another existing bug in `sct_propseg`. 

Because this block of code has been silently evaluating to `True` in all cases, we've ended up incorrectly relying on it to move the output files to the output directory. So, now that this block of code is (correctly!) skipped, it has the side effect of no longer moving the output files as expected. (Hence the FileNotFoundError in the failing test.)

So, I believe the `mv` commands have to be placed outside of this block, so that they execute as they did before.

```suggestion
    mv(os.path.join(folder_output, add_suffix(os.path.basename(fname_data_propseg), "_seg")),
               fname_seg)
    mv(os.path.join(folder_output, add_suffix(os.path.basename(fname_data_propseg), "_centerline")),
    if rescale_header != 1:
```

Reply:
In fact, I'm not even sure if this `if rescale_header` check is even doing anything related to rescaling.